client/src/pages/Profile.tsx

```diff
@@ -26,50 +25,49 @@ const Profile = () => {
         console.error('Error fetching user data: ', error);
       });
 
-    fetch('/api/solutions/1')
-      .then(response => {
-        if (!response.ok) {
-          throw new Error('Failed to fetch solution data');
-        }
-        return response.json();
-      })
-      .then(data => {
-        setSolutions(data);
-      })
-      .catch(error => {
-        console.error('Error fetching solution data:', error);
-      });
+    // fetch('/api/solutions/1')
```

Comment:
I will likely finish the profile page this week. I have talked to Alex about this already. 

It will likely be in the following prs. For now, I just minimized the profile page so it doesn't throw any errors.